src/annotator.cpp

```diff
@@ -450,30 +446,33 @@ void Annotator::AnnotatorImpl::addIssueNoModel() const
 void Annotator::AnnotatorImpl::addInvalidArgument(CellMLElement type) const
 {
     auto issue = Issue::create();
-    issue->setDescription("The given type '" + cellMLElementAsString(type) + "' is invalid for this method.");
+    auto description = "The item is internally inconsistent: the enum type '" + cellmlElementAsString(type) + "' cannot be used with the stored item.";
+    issue->setDescription(description);
     issue->setLevel(Issue::Level::ERROR);
     mAnnotator->addIssue(issue);
 }
 
 bool Annotator::AnnotatorImpl::exists(const std::string &id, size_t index) const
 {
-    bool exists = true;
     if (!mAnnotator->hasModel()) {
         addIssueNoModel();
-        exists = false;
-    } else if (mIdList.count(id) <= index || mIdList.count(id) == 0) {
+        return false;
+    }
+    if ((mIdList.count(id) <= index) || (mIdList.count(id) == 0)) {
         addIssueNotFound(id);
-        exists = false;
+        return false;
     }
-
-    return exists;
+    return true;
 }
 
 AnyItem Annotator::item(const std::string &id)
 {
     AnyItem retrieved = std::make_pair(CellMLElement::UNDEFINED, nullptr);
-    if (isUnique(id)) {
+    auto num = duplicateCount(id);
```

Comment:
hasn't `duplicateCount` been renamed?

Reply:
Nope, see https://github.com/cellml/libcellml/pull/640/files#r499497547. Some of use are fine with renaming it `itemCount`. So, am happy to make that change. Agreed?

Reply:
Done in commit 47839635.